godtools/App/Services/Renderer/Views/MobileContent/Views/ContentText/MobileContentTextViewModel.swift

```diff
@@ -126,14 +134,8 @@ class MobileContentTextViewModel: MobileContentViewModel {
     }
     
     private func getFontWeight() -> UIFont.Weight {
-             
-        let textStyles: Set<Text.Style> = textModel.textStyles
-        
-        if textStyles.contains(.bold) {
-            return .bold
-        }
         
-        return .regular
+        return isBold ? .bold : .regular
```

Comment:
Not in the scope of this PR, but I've deprecated `TextStyle.Bold` in favor of `text.fontWeight`.

in the shared library I handle the fallback of bold -> font-weight for you, so once you implement support for `font-weight` you will no longer need to check for the `bold` text style

Reply:
I had wanted to handle the raw font-weight to `UIFont.Weight` conversion in the shared module for you, but I couldn't find the `UIFont.Weight` constants when I looked

Reply:
Hey thanks @frett.  Do you know if there's a ticket for upgrading to use ```text.fontWeight```?

Reply:
there is tickets created for it yet, the only use case for it currently is web. And I don't have an easy implementation path on Android with the current renderer views.